osmpbfreader.h

```diff
@@ -192,7 +192,9 @@ struct Parser {
             if(sz != blob.raw_size())
                 warn() << "  reports wrong raw_size: " << blob.raw_size() << " bytes";
 
-            memcpy(unpack_buffer, buffer, sz);
+			//memcpy(unpack_buffer, buffer, sz);
+			memcpy(unpack_buffer, blob.raw().c_str(), sz); // buffer is not the raw data
```

Comment:
The comments are useless outside this PR, please remove them. Your also need to indent with spaces only.